lib/tasks/census_api.rake

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+#
+# A set of utils to manage and validate verification related data.
+#
+DOCUMENT_TYPE = {
```

Comment:
This constant already exist in DigraCensusAuthorizationHandler. You can use the constant in there instead of redefining the same constant here..